src/rcms/fm/app/level1/HCALMasker.java

```diff
@@ -232,7 +232,9 @@ else if (functionManager.RunType.equals("global")){
               allMaskedResources = (VectorT<StringT>)functionManager.getHCALparameterSet().get("MASKED_RESOURCES").getValue();
               for (Resource level2resource : fullconfigList) {
                 logger.debug("[HCAL " + functionManager.FMname + "]: The masked level 2 function manager " + qr.getName() + " has this in its XdaqExecutive list: " + level2resource.getName());
-                allMaskedResources.add(new StringT(level2resource.getName()));
```

Comment:
In here, we are adding all childResources of any maskedFM to the `allMaskedResources` (see my comment few lines above)